src/VolleyManagement.API/Startup.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Controllers;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using SimpleInjector;
+using SimpleInjector.Integration.AspNetCore.Mvc;
+using SimpleInjector.Integration.WebApi;
+using SimpleInjector.Lifestyles;
+using System.Threading.Tasks;
+using System.Web.Http;
+using VolleyManagement.API.Infrastructure;
+using VolleyManagement.Crosscutting.IOC;
+using VolleyManagement.Data.MsSql.Context;
+using VolleyManagement.Data.MsSql.Infrastructure;
+using VolleyManagement.Services.Infrastructure;
+
+namespace VolleyManagement.API
+{
+    public class Startup
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this class into smaller and more specialized ones to reduce its dependencies on other classes from 31 to the maximum authorized 20 or less. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1200)

Reply:
Suppress it